pieo-trees/notes.tex

```diff
@@ -225,19 +225,20 @@ \section{Structure \& Semantics}
 \section{Well-Formedness}
 
 \begin{dfn}
-    Define $|\cdot| : \PIEOTree(t) \to \mathbb N$ by
+    Fix $f \in \mathcal F$.
+    Define $|\cdot|_f : \PIEOTree(t) \to \mathbb N$ by
     \begin{align*}
-        |\Leaf(p)| = |p| && |\Internal(qs, p)| = \sum_{i=1}^{|qs|} |qs[i]|
+        |\Leaf(p)| = |p|_f && |\Internal(qs, p)| = \sum_{i=1}^{|qs|} |qs[i]|_f
```

Comment:
Aha, thanks! Sorry I didn't catch this earlier, but yeah we'd need to do the @sampsyo-suggested PIEO-to-PIFO filtering before making any comment on numbers of members!